Flutter/pdf-viewer/text-selection.md

```diff
@@ -147,11 +169,13 @@ Widget build(BuildContext context) {
 {% endhighlight %}
 {% endtabs %}
 
-## How to create and display a customized text selection context menu with a Copy option to retrieve the selected text?
+## How to create and display a customized text selection context menu with a Copy and Text markup options to retrieve the selected text?
```

Comment:
Please lower case this.

<img width="378" alt="image" src="https://github.com/syncfusion-content/flutter-docs/assets/65169077/e1f9989a-930a-4a20-b396-b959e6890139">

Reply:
Corrected the case